docs/lightning/02-installation/06-adapter-installation-troubleshooting.md

```diff
@@ -28,8 +28,8 @@ This can happen if the customer has checked “Enable clickjack protection” on
 <img src={useBaseUrl('/img/lightning/troubleshooting4.png')} />
 
 These are options we provide that allow you to toggle certain functionality in the adapter.
-- CCA Case Trigger - This trigger looks for any ContactChannelAnalytics records that could be related to a updated/inserted Case, and creates a relationsihp between the two records.
-- CCA Contact Trigger - This trigger looks for any ContactChannelAnalytics records that could be related to a updated/inserted Contact, and creates a relationsihp between the two records.
+- CCA Case Trigger - This trigger looks for any ContactChannelAnalytics records that could be related to a updated/inserted Case, and creates a relationsihp between the two records. This trigger uses batching to process the update requests.
+- CCA Contact Trigger - This trigger looks for any ContactChannelAnalytics records that could be related to a updated/inserted Contact, and creates a relationsihp between the two records. This trigger uses batching to process the update requests.
```

Comment:
do we need to add this to the classic docs or no?

Reply:
It looks like this entire section isn't on the classic docs - not sure if we should add it to it or not